src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -550,10 +551,24 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
         `ui.panel.config.integrations.config_entry.state.${item.state}`,
       ];
       if (item.reason) {
-        this.hass.loadBackendTranslation("config", item.domain);
-        stateTextExtra = html`${this.hass.localize(
-          `component.${item.domain}.config.error.${item.reason}`
-        ) || item.reason}`;
+        const lokalisePromError = this.hass
+          .loadBackendTranslation("config", item.domain)
+          .then((localize) =>
+            localize(`component.${item.domain}.config.error.${item.reason}`)
+          );
```

Comment:
Why do we load this when we are only gonna use it if there is no `error_reason_translation_key`? Why not load it in the else?